release_notes/ocp-4-15-release-notes.adoc

```diff
@@ -1496,6 +1496,8 @@ In the following tables, features are marked with the following statuses:
 
 * Specifying a standard Ebdsv5 or Ebsv5 family machine type instance is not supported when installing an Azure cluster. This limitation is the result of the Azure terraform provider not supporting these machine types. (link:https://issues.redhat.com/browse/OCPBUGS-18690[*OCPBUGS-18690*])
 
+* When running a cluster with FIPS enabled, you might receive the following error when running an older version of the OpenShift CLI (`oc`) on a {op-system-base} 8 or {op-system-base} 9 system: `FIPS mode is enabled, but the required OpenSSL backend is unavailable`. As a workaround, use the `oc` binary provided with the {product-title} cluster. (link:https://issues.redhat.com/browse/OCPBUGS-18690[*23386-23386*])
```

Comment:
What about this, since oc works on RHEL8;

```suggestion
* When running a cluster with FIPS enabled, you might receive the following error when running an older version of the OpenShift CLI (`oc`) on a {op-system-base} 9 system: `FIPS mode is enabled, but the required OpenSSL backend is unavailable`. As a workaround, use the `oc` binary provided with the {product-title} cluster. (link:https://issues.redhat.com/browse/OCPBUGS-18690[*23386-23386*])
```

Reply:
Thanks, updated!